gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/internal/configureApplication.kt

```diff
@@ -94,6 +94,20 @@ internal fun Project.configurePackagingTasks(apps: Collection<Application>) {
             into(destDir)
         }
 
+        val preparePackagingResources = tasks.composeTask<Sync>(
+            taskName("preparePackagingResources", app)
+        ) {
+            val packagingResourcesRootDir = app.nativeDistributions.packagingResourcesRootDir
+            if (packagingResourcesRootDir.isPresent) {
+                from(packagingResourcesRootDir.dir("common"))
+                from(packagingResourcesRootDir.dir(currentOS.id))
+                from(packagingResourcesRootDir.dir(currentTarget.id))
+            }
+
+            val destDir = project.layout.buildDirectory.dir("compose/tmp/resources")
+            into(destDir)
+        }
+
```

Comment:
This allows to specify different files per OS/arch combination, just like with `appResourcesRootDir`.